jaseci_core/jaseci/api/architype_api.py

```diff
@@ -20,7 +20,7 @@ class architype_api():
     a Jaseci instance is aware of.
     """
 
-    @interface.private_api()
+    @interface.private_api(cli_args=['code'])
```

Comment:
Can I have multiple `cli_args`? I'd assume no because then their position would matter, right?

Reply:
You can have multiple, just would need to keep in mind whatever is in the list will have to be the first args to the given command in that order then you'd use the `-param {value}` in any order for other params.